libtransmission/rpc-server.cc

```diff
@@ -1183,42 +1196,36 @@ tr_rpc_server::tr_rpc_server(tr_session* session_in, tr_variant* settings)
     if (!tr_variantDictFindStrView(settings, key, &sv))
     {
         missing_settings_key(key);
-        address = tr_rpc_inaddr_any;
+        bindAddress->inaddr_any();
```

Comment:
By its name, this looks like a getter call with unused result. Maybe change to `set_inaddr_any()`?..